src/appengine/private/components/testcase-detail/testcase-detail.html

```diff
@@ -727,6 +729,10 @@
         return this.isStatusReady(info) &&
             !this.isNA(info.testcase.minimized_keys);
       },
+      shouldShowVariants(info) {
+        return !info.testcase.one_time_crasher_flag &&
+            info.testcase.minimized_keys;
```

Comment:
Should this be `&& !info.testcase.minimized_keys`?
I assume this code should be returning false if no minimized testcase is available right?
What if minimization errors out?

Reply:
if minimization fails, it calls mark_unreproducible_if_flaky, which sets one_time_crasher_flag to true.
and it should be "info.testcase.minimized_keys" since variant pending task are only created after minimization is finished.

Reply:
ACK